curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64649c6d77059edf80279a32.md

```diff
@@ -0,0 +1,193 @@
+---
+id: 64649c6d77059edf80279a32
+title: Step 95
+challengeType: 0
+dashedName: step-95
+---
+
+# --description--
+
+Next, use string interpolation (`${}`) to set the `id` attribute to the `inputVal` property of the current object, `obj`.
```

Comment:
```suggestion
Next, use string interpolation to set the `id` attribute to the `inputVal` property of the current object, `obj`.
```

This is the seventh project using template literals - shouldn't need an example.